### DISCUSSION TITLE
Default email address per repo/organisation

### DISCUSSION THREAD

**Discussion by marksweb** [community/community#55059] [Repositories]
### Select Topic Area

Product Feedback

### Body

I'm sure I'm not the only person who has multiple email addresses on their account and a member of multiple organisations.

When you're carrying out actions on the site, like updating a PR for example, github will associate the commit with one of your email addresses. I think it takes the default, but it seems to choose others for me from time-to-time.

It'd be really good if this email address choice was a select box from the available addresses. Or even something you could define in your account settings, where you say which email address should be used in the organisations/repos you have write access to.

As an example scenario, I've just updated a PR in my work organisation which has been committed with my personal email account. But I've got my work email available as well, which would have been much more suited as it's what I commit from my machine with. The same occurs with an open source organisation I work with where I have an email account for their domain.

**Comment by atharen** [comment]
[Might have found a solution here](https://github.com/todogroup/gh-issues/issues/32#issuecomment-1092800544) (_but at the top of the page now..._), which maybe could also be made visible from the organisation itself (more of a UX issue).

There might still be some caveats as noted from the replies, but for now I'm just settling on that, and see how it works out...
